anuvaad-etl/anuvaad-extractor/file_translator/etl-file-translator/services/service.py

```diff
@@ -113,18 +115,33 @@ def get_runs(self, iterable_obj, para_obj=False, run_obj=False, run_lst=False, f
         if para_obj and file_type == config.TYPE_PPTX:
             return iterable_obj.runs
         if para_obj:
-            runs = []
+            runs = [[] for i in range(20)]
+            run_list_counter = 0
             for rid, child in enumerate(iterable_obj._element):
                 if isinstance(child, CT_R):
                     run_obj = Run(child, iterable_obj)
-                    runs.append(run_obj)
+                    runs[run_list_counter].append(run_obj)
+                elif isinstance(child, CT_MATH):
```

Comment:
have you tested for the scenario where there are two consecutive math formulas (two math tags) or 1 math and 1 drawing tag one after the other.
If yes, the output was correct?

Reply:
Yes it was for this reason that I initialized a list of lists earlier so that empty lists generated by consecutive math/drawing other special tags will not cause issues.

Reply:
but as we discussed we will not create a 2d list beforehand, i think you are still creating a 20 blank list beforehand , so fix that and handle the case which I have mentioned above.